build.groovy

```diff
@@ -268,14 +269,15 @@ options:
 	cli.r(longOpt:'reset', 'Deletes the dependency collections and build result group from the MetadataStore')
 	cli.v(longOpt:'verbose', 'Flag to turn on script trace')
 	cli.pv(longOpt:'preview', 'Supplemental flag indicating to run build in preview mode without processing the execute commands')
+	cli.cds(longOpt:'checkSystemDatasets', 'Optional flag to validate the presense of the defined system datasets. ')
```

Comment:
Shouldn't it be `-csd` as the first letters of "Check System Datasets"?

Reply:
@M-DLB - `CSD` is already a known term from CICS. Should we rename the long option to `checkDatasets` instead ? 

I added the _system_ datasets, because it is really just the system datasets that can be validated in advance and not the additional build datasets that are created during build time.

Reply:
Let's call it `checkDatasets `then, so `-cd` should be fine :)

Reply:
Hi,
Or -ivp (Installation Verification Procedure)?